addons/website/tests/test_ui.py

```diff
@@ -110,7 +110,10 @@ def test_admin_tour_rte_translator(self):
 @odoo.tests.common.tagged('post_install', '-at_install')
 class TestUi(odoo.tests.HttpCase):
 
-    def test_01_restricted_editor(self):
+    def test_01_admin_tour_homepage(self):
+        self.start_tour("/?enable_editor=1", 'homepage', login='admin', step_delay=100)
```

Comment:
Is the step delay actually useful ?

Reply:
I made a quick check and at first glance, it doesn't seem like a good idea to add in tests (if you check [why it was made optional](https://github.com/odoo/odoo/commit/6f7f235799128dc650b3f94994aa9a1a55a1f033)). But No, I think it's not useful here.

Reply:
Yes, it's of course ideal to remove it. But sometimes, some people did add them in tests to hide bug instead of fixing them... let's suppose our editor is robust enough now and remove it :)